### PR TITLE
Update sign in/up UI - Closes #4306

### LINKED ISSUE #4306: Update sign in/signup UI
### Description

Please describe the specification of new features/improvements

### Acceptance Criteria

- Modify PassphraseBackup Component yo match the new  [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2070)
- Update passphraseBackup Screen to match with the [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2070)
    - recovery phrase pdf download needs to be remove
    - title and description need to be update
-  Update confirmPassphrase screen to match with the [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2122)
- Update accountCreated Screen to match with the [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2184)
- update sign in screen to match with the [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2317)
- adopt unit test

### PR DESCRIPTION
**Pull request by isalga**
### What was the problem?

This PR resolves #4306

### How was it solved?

Update design

### How was it tested?

Visually

### CODE REVIEW

**Comment by eniolam1000752** [comment]
You've got a failing CI at `unit test` stage due to a lower coverage value @isalga

**Review by reyraa — DISMISSED** [review]
- The copy icon looks different. Can we add the correct one here?
- I can see the sign in button in the register screen, which is fine. But it's missing in the design. Can we get an approval from @ann-vashchenko ?
- Also, the network is visible at the top. But registering an account is not network-specific. I think we should hide it. As it's hidden in the design too. Correct @ann-vashchenko ?
- The color of the subtitle, and the corner raius of the option buttons are different than in the design of the [design](https://www.figma.com/file/GbrsJiZlsh2OzVKqYDD4zI/Account-management-improvements?node-id=1%3A2122) The subtitle color has a similar issue in the add-account page too.
- There's a delay when the secret is approved. The options turn green, we don't navigate to the next step, but the primary button is still active. should we disable it? should we navigate right away? What is the delay for?

**Comment by isalga** [comment]
@reyraa, I discussed with @ann-vashchenko and she mentions sign in button has to be removed. For the third point (network) it is out of the scope of this ticket and it needs discussion with @ManuGowda . Right @ann-vashchenko ?

**Review by gadjacobs — DISMISSED** [review]

**Review by soroushm — DISMISSED** [review]

File: src/modules/common/components/bars/topBar/topBar.js [hunk 1]
```diff
@@ -62,12 +60,9 @@ const TopBar = ({
       <div className={styles.group}>
         <LightDarkToggle />
         {!isUserLogout && <DiscreteModeToggle />}
-        <Network token={token.active} network={network} t={t} />
-        {isUserLogout && history.location.pathname !== routes.login.path ? (
-          <Link to={routes.login.path} className={styles.signIn}>
-            <PrimaryButton size="s">Sign in</PrimaryButton>
-          </Link>
-        ) : null}
```
Comment (soroushm) [inline src/modules/common/components/bars/topBar/topBar.js]: @isalga please revert this sign in button right now application have braking change, this one is not blong to this ticket

**Review by soroushm — APPROVED** [review]

**Review by gadjacobs — APPROVED** [review]

**Pull request merged**